.release-notes/44.md

```diff
@@ -0,0 +1,3 @@
+## Add support for not outputting color to terminal
+
+This is required when the output is piped into a file or a program.
```

Comment:
Can you expand on this noting that prior to this change, any error formatted for output to the console would always be colorized and then API has been changed to allow for it not to be because of piping.

Note that it is a non-breaking change as it will default to the old behavior and then give a code snippet to show how to turn off colorization.

Reply:
Expanded, hopefully it is enough